mslice/models/cut/matplotlib_cut_plotter.py

```diff
@@ -20,7 +20,6 @@ def plot_cut(self, selected_workspace, cut_axis, integration_start, integration_
         leg.draggable()
         plt.xlabel(self._getDisplayName(cut_axis.units, self._cut_algorithm.getComment(selected_workspace)))
         plt.ylabel(INTENSITY_LABEL)
-        plt.gcf().canvas.set_window_title(selected_workspace)
```

Comment:
by default, the cut plots have no title. I was just sticking to this, so it would be figure 1 by default. Can change it if you would prefer.